src/indra_cogex/client/queries.py

```diff
@@ -1140,6 +1127,345 @@ def get_stmts_for_stmt_hashes(
     }
     return rv, evidence_counts
 
+@autoclient()
+def get_statements(
+        agent: Union[str, Tuple[str, str]],
+        *,
+        rel_types: Optional[Union[str, List[str]]] = None,
+        stmt_sources: Optional[Union[str, List[str]]] = None,
+        agent_role: Optional[str] = None,
+        other_agent: Optional[Union[str, Tuple[str, str]]] = None,
+        other_role: Optional[str] = None,
+        paper_term: Optional[Tuple[str, str]] = None,
+        mesh_term: Optional[Tuple[str, str]] = None,
+        include_child_terms: Optional[bool] = True,
+        limit: Optional[int] = 10,
+        client: Neo4jClient,
```

Comment:
I think the ordering here works because of the autoclient decorator, but we should still move the `client` keyword argument to before the optional ones.